### ISSUE TITLE
NTR: Ingestion of undercooked pork from areas with cysticercosis (Immune system assessed clinically)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8045]
New term request Ingestion of undercooked pork from areas with cysticercosis (segal_090804191533):   UMLS: 2107846

**Comment by pnrobinson** [comment]
@MickeySegal
I would think Recent history of undercooked pork ingestion would be a good HPO term, but I would not include the part about areas with cysticercosis -- this bundles too many concepts and would complicate use of the term.
Thoughts?

**Comment by MickeySegal** [comment]
If someone ate pork from the USA I'd have low suspicion of cysticercosis, but eating pork from countries with less careful health standards would elicit much more suspicion.  So the location is very important information, but I don't know how best to represent it with no bundling.

**Comment by pnrobinson** [comment]
@MickeySegal 
Probably this finding will only be elicited if the patient comes from an area with cysticercosis. For conciseness sake and also so as to avoid the question of what exactly constitutes an an area with cysticercosis, I formulated like this. Please reopen if needed.


Adding new term: 
- ID: HP:6000727
- label: History of undercooked pork ingestion
- parents: [Nutrition history (HP:4000123)](https://hpo.jax.org/app/browse/term/HP:4000123)
- pmid: PMID:23265549
- def: A history of ingestion of undercooked pork. This is particularly relevant in areas with a high prevalence of cysticercosis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8045
- synonyms:
